w3c-ruleset-review-process.md

```diff
@@ -0,0 +1,88 @@
+
+### Survey
+
+A facilitator of the ACT Task Force creates a survey for the rule, in which Task Force members are asked the following questions:
+
+- Does the rule follow the ACT Rules Format 1.0?
+- Are the assumptions acceptable?
+- Is the implementation data correct?
+- Is the rule consistent with existing WCAG documents?
+- Are there any remaining open issues for this rule that were opened prior to this review?
+- Do you have any further questions or concerns about this rule?
+- Do you think this rule is ready to be published?
+
+This survey will be open for at least 5 work days, to give ACT Task Force members sufficient time to review the rule. No more than 3 rule surveys will be open at the same time.
+
+If the rule is rejected during the review, one of the ACT facilitators sends the feedback to the rule author. The rule author will have 5 work days to correct any reported issues. If not, the issue will be closed. Issues can always be reopened, which will restart the process from step 1. This puts the proposal on the bottom of the backlog.
+
+### Publication
+
+When a rule passed the review stage, it is included in a draft version of the WCAG Ruleset. Depending on the types of changes, and the availability of the AG Working Group, the ACT Task Force decided on a timeline for releasing updates to the WCAG Ruleset. To update the WCAG Ruleset, the ACT Task Force requests approval from the AG Working Group.
+
+## Implementations
+
+An implementation is either an accessibility testing tool, or a testing methodology. Implementations show that the ACT rule works in practice, and that the interpretation made by the rule is one that is used in real-world accessibility testing. Because of this, having more implementations, or having an implementation that has a lot of users weighs heavily in demonstrating that the assumptions and interpretation in the rule are acceptable to the accessibility community.
+
+For an implementation to be considered, it must be "correct". This is determined by running the tool or methodology on the test cases provided in the rule. Acknowledging that not all implementors distinguish between "passed" and "inapplicable", and that semi-automated tools can return "cantTell" as a result, an implementation is considered correct when it gets one of the allowed outcomes for all of the test cases:
+
+| Test Case Type | Allowed outcomes                        |
+| -------------- | --------------------------------------- |
+| Passed         | `passed`, `cantTell`, or `inapplicable` |
+| Failed         | `failed` or `cantTell`                  |
+| Inapplicable   | `inapplicable`, `cantTell`, or `passed` |
+
+## Public Feedback
```

Comment:
This seems like a separate process to me, or a "fork" of the section "Proposing And Updating A Rule". However, this is not at all clear from the structure of this document.